data/Psi4/basicPsi4-1.2.1/water_mp2.out

```diff
@@ -1,9 +1,9 @@
 
     -----------------------------------------------------------------------
           Psi4: An Open-Source Ab Initio Electronic Structure Package
-                               Psi4 1.2.1 release
+                               Psi4 1.3.2 release
```

Comment:
You've accidentally run Psi4 1.3.2 in the folder for 1.2.1. 1.2.1 should be available from their conda channel, so you can make a new env with that version.

Reply:
Yes. I'm fetching 1.2.1 from conda now. Thanks.